wiki/content/query-language/index.md

```diff
@@ -2183,6 +2188,41 @@ Queries using the `expand(_all_)`, `expand(_reverse_)`, or `expand(_forward_)`
 functions now require that the types of the nodes to expand have been properly
 set. Refer to that section for more information.
 
+In addition, reverse edges must be specified in the schema type in both directions. The reverse edge can be an alias ​​or even the same value as the outcoming edge. Also queries with `expand(_all_)` and `expand(_reverse_)` will be expanded normally.
+
+{{% notice "note" %}} Note: You do not have to specify a predicate for "author" inside Article Type because this is an alias.{{% /notice  %}}
+
+E.g.
+```
+type Author {
+  name: string
+  age: int
+  nationality: string
+  articles: [Article]
```

Comment:
Why `author` is a reverse edge and not `articles`?
and either we should use `author*s*: [Author]` or `article: [Article]`, should be consistent.